makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+.PHONY: all
+
+all: code-format sanity-check unit-test
+
+# TODO: install those dependencies in docker image (dockerfile).
+install-ci-dependency:
+	bash tools/ci_build/install/install_ci_dependency.sh --quiet
```

Comment:
@seanpmorgan Perhaps we need to build own images for addons in the future?

Reply:
Have no issue with doing this, but one thing we'll need to figure out is what account will host it on dockerhub? Or do you imagine just a Dockerfile in the repo that should be built by the user?

Reply:
I prefer to host it on dockerhub, like https://hub.docker.com/r/tensorflow/tensorflow/tags/ . And both our CI and contributors can use it to run check / test /build. What do you think? Do we need to create an issue to discuss it?

Reply:
I like the idea, but yeah probably need another issue to discuss. For example, someone on the TF team may need control the Dockerfile+Build if they're publishing under their tag.